llvm/test/MC/RISCV/corev/simd/absb-invalid.s

```diff
@@ -0,0 +1,21 @@
+# RUN: not llvm-mc -triple=riscv32 --mattr=+xcorevsimd %s 2>&1 \
```

Comment:
Consider merging both the invalid and valid tests into one. This will halve the number of files needed. Reducing the number of tests will make it easier to review this patch. This will also be more considerate to upstream developers.
i.e. consider merging `absb-invalid.s` and `absb.s`. This way you will only have one complete test for each instruction which tests for both valid and invalid uses of the instruction. I suggest simply calling the new test `absb.s`.

Reply:
Yes I agree the number of tests could be reduced. There are so many tests because having individual tests for each instruction helps locate the problematic instruction when I was developing the patch.

Reply:
Number of tests is reduced, so we can resolve this issue.